### PR TITLE
Fix#396 - Rename tutorials + Fix names in sidebar

### LINKED ISSUE #396: Renaming the file paths to avoid issues on Windows
## Since Windows does not allow colons in file names, I think we need to rename these files in Git to remove or replace the colons.

Initially, I intended to fix it and create a Pull Request. However, these files cannot be displayed on my local Windows OS. 
@Ash-2k3 Could you please take a look and help me resolve these issues? Thanks a lot!
(/cc @U8NWXD for information)

### Specifically, the following files:

1. **Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md** 
2. **Tutorial:-Learn-to-figure-out-the-reproduction-steps-for-server-error.md** 

error_window

### Suggestions:
1. Rename the files using the `git mv` command:
   - ```bash
     git mv "Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md" "Tutorial-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md" 
     ```
   - ```bash
     git mv "Tutorial:-Learn-to-figure-out-the-reproduction-steps-for-server-error.md" "Tutorial-Learn-to-figure-out-the-reproduction-steps-for-server-error.md" 
     ```

2. Update the content in the `_Sidebar.md` file without colon:
   - `[[Tutorial - Learn-to-figure-out-the-reproduction-steps-for-server-error|Tutorial-Learn-to-figure-out-the-reproduction-steps-for-server-error]]`
   - `[[Tutorial - Learn to write and test a non-trivial beam job|Tutorial-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job]]`

### PR DESCRIPTION
**Pull request by Ash-2k3**
Fix https://github.com/oppia/oppia-web-developer-docs/issues/396

Rename tutorials - Remove colons from the name of the tutorials

Fix the tutorial links name in Sidebar.

### CODE REVIEW

File: Tutorial-Learn-How-to-Make-a-Simple-UI-Change.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Tutorial: Making a Simple UI Change
```
Comment (seanlip) [inline Tutorial-Learn-How-to-Make-a-Simple-UI-Change.md:1]: You've deleted the header of the page, so what is the page going to look like on the GitHub wiki? Will readers know what the page refers to?

File: _Sidebar.md [hunk 2]
```diff
@@ -52,7 +52,7 @@
     * [[Debug frontend code|Debug-frontend-code]]
     * [[Debugging Stories|Debugging-Stories]]
     * [[Testing email functionality|Testing-email-functionality]]
-    * [[Tutorial - Learn-to-figure-out-the-reproduction-steps-for-server-error|Tutorial:-Learn-to-figure-out-the-reproduction-steps-for-server-error]]
+    * [[Tutorial - Learn to figure out the reproduction steps for server error|Tutorial-Learn-to-Figure-Out-the-Reproduction-Steps-for-Server-Error]]
```
Comment (seanlip) [inline _Sidebar.md]: for a server error

**Review by seanlip — CHANGES_REQUESTED** [review]

File: Tutorial-Learn-How-to-Make-a-Simple-UI-Change.md (hunk 1 above)
Comment (Ash-2k3) [reply Tutorial-Learn-How-to-Make-a-Simple-UI-Change.md:1]: Github wiki automatically renders the file name as the top level header (I learnt about it just recently) - https://github.com/oppia/oppia/wiki/Making-a-simple-UI-change. If you checkout the wiki page, you could see two headers which is making it look a bit odd.

File: _Sidebar.md (hunk 2 above)
Comment (Ash-2k3) [reply _Sidebar.md]: Done, thanks!

**Review by seanlip — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**